couchbase/kv/v1/kv.proto

```diff
@@ -397,6 +383,20 @@ message MutateInResponse {
   MutationToken mutation_token = 3;
 }
 
+message GetAllReplicasRequest {
+  string bucket_name = 1;
+  string scope_name = 2;
+  string collection_name = 3;
+  string key = 4;
+}
+
+message GetAllReplicasResponse {
+  bool is_master = 1;
```

Comment:
is_primary?

Reply:
I think they call these the 'active vs replica' vbuckets, so maybe `is_active`?

Reply:
Oh wait, maybe we go the other way and have it be 'is_replica' instead.